datalab/datalab_session/tasks.py

```diff
@@ -19,6 +21,6 @@ def execute_data_operation(data_operation_name: str, input_data: dict):
     else:
         try:
             operation_class(input_data).operate()
-        except Exception as e:
-            log.error(f"Error executing {data_operation_name}: {type(e).__name__}:{e}")
+        except ClientAlertException as e:
+            log.error(f"Client Error executing {data_operation_name}: {type(e).__name__}:{e}")
```

Comment:
I'm a little afraid that if we get some 3rd party library exception we are not expecting, it will feel bad for the user since it won't be caught anymore. Could you leave the general `except Exception` case in there as well and just log it plus set the error for the frontend to something like "Uknown error occurred, please try again and report to the development team if it persists"